.github/workflows/sketch-compile.yml

```diff
@@ -0,0 +1,53 @@
+name: Sketch Compile
+
+# the action is triggered at every pull request
+on:
+  - pull_request
+
+jobs:
+  sketch-compile:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        fqbn:
+          - arduino:samd:mkrwifi1010
+          - arduino:samd:nano_33_iot
+          - arduino:megaavr:uno2018:mode=on
+          - arduino:mbed:nano33ble
+          - arduino:mbed_nano:nanorp2040connect
+          - arduino:renesas_uno:unor4wifi
```

Comment:
should we remove these boards and use only the one we're working with?

Reply:
@Frack91

Reply:
I'd prefer to keep the wemos configuration and remove everything else for now, as it's the only board we're currently supporting.